ompi/mpi/fortran/use-mpi-ignore-tkr/mpi-ignore-tkr.F90

```diff
@@ -42,6 +43,9 @@ module mpi
 ! The ignore-TKR version of the MPI interfaces
 
   include "ompi/mpi/fortran/use-mpi-ignore-tkr/mpi-ignore-tkr-interfaces.h"
+#if OMPI_FORTRAN_HAVE_BIND_C_TYPE
```

Comment:
I think you need to add `OMPI_FORTRAN_HAVE_BIND_C_TYPE` to `ompi/mpi/fortran/configure-fortran-output.h.in`, because right now this will always be false.